engine/src/main/resources/assets/schemas/schemaSolNames.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Names for in-game systems/planets, selected randomly in world generation",
+  "propertyNames": {
+    "pattern": "^[\\w\\ ]*$",
```

Comment:
In the description you state planet names can't be empty. Here the `.` will match zero or more, meaning an empty string is a valid match here.

Furthermore, what's the `\ ` for? You can just have space unless I'm missing something

Reply:
Fixed to remove the empty string match
The extra slash was my mistake, it has also been removed.